etc/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+# Tack version onto and install the bash completion scripts
+
+# Install script to source all bash completion scripts, of all library versions.
+configure_file(
+  "gz.completion"
+    "${CMAKE_CURRENT_BINARY_DIR}/gz${PROJECT_MAJOR_VERSION}.completion" @ONLY)
+install(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/gz${PROJECT_MAJOR_VERSION}.completion
+  DESTINATION
+    ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}/gz)
+
+configure_file(
+  "ign.bash_completion.sh"
```

Ah final request, can this be 

```suggestion
  "gz.bash_completion.sh"
```

---

That file is tied to this installation line in the release repo, and it would need to be changed there too: https://github.com/gazebo-release/gz-tools-release/pull/4

Do we want to change it in the release repo?